### PR TITLE
[release-4.14] OCPBUGS-21737: Bump deps to address CVE-2023-44487

### PR DESCRIPTION
**Pull request by knelasevero**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -113,3 +113,13 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace (
+	// To address CVE-2023-44487 until k8s 1.27.5 release is out
```
Comment (ingvagabund) [inline go.mod:118]: 1.27.7

Comment (knelasevero) [reply go.mod]: :heavy_check_mark:

**Pull request merged**